Client/control/keyboard.py

```
@@ -0,0 +1,24 @@
+import pyautogui
+import logging
+
+def type_text(text, interval=0.05):
+    try:
+        pyautogui.write(text, interval=interval)
+    except Exception as e:
+        logging.error(f"Error in type_text: {e}")
```

Ruff (0.15.0)

[warning] `error()` call on root logger

(LOG015)